internal/backend/remote-state/consul/client_test.go

```diff
@@ -211,19 +212,11 @@ func TestConsul_largeState(t *testing.T) {
 
 	// We use a fixed seed so the test can be reproductible
 	randomizer := rand.New(rand.NewSource(1234))
-	RandStringRunes := func(n int) string {
-		var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
-		b := make([]rune, n)
-		for i := range b {
-			b[i] = letterRunes[randomizer.Intn(len(letterRunes))]
-		}
-		return string(b)
-	}
 
 	testPayload(
 		t,
 		map[string]string{
-			"bar": RandStringRunes(5 * (524288 + 2)),
+			"bar": testutils.RandomIDFromSource(randomizer, 5*(524288+2), testutils.CharacterRangeAlpha),
```

Comment:
I haven't looked closely at your style guide, but another PR had `//nolint:mnd` (or something similar). And `524288` looks like a magic number to me.